package.json

```diff
@@ -54,9 +54,11 @@
     "@patternfly/patternfly": "4.135.2",
     "@patternfly/react-core": "4.157.3",
     "@patternfly/react-styles": "4.11.16",
+    "@patternfly/react-icons": "4.11.17",
```

Comment:
That is fully correct -- it's a dependency anyway, and we really ought to track the version explicitly. This has bitten us in cockpit and other projects already, as not all versions are compatible with each other.